src/CourseAuthoringPage.jsx

```diff
@@ -57,7 +57,7 @@ const CourseAuthoringPage = ({ courseId, children }) => {
       we shouldn't have the header and footer on these pages.
       This functionality will be removed in TNL-9591 */}
       {inProgress ? !pathname.includes('/editor/') && <Loading />
-        : <AppHeader courseNumber={courseNumber} courseOrg={courseOrg} courseTitle={courseTitle} courseId={courseId} />}
+        : AppHeader(courseNumber, courseOrg, courseTitle, courseId)}
```

Comment:
@BilalQamar95 Rendering a React component by calling it as a function instead of rendering it with JSX (i.e., `<AppHeader ... />`) is an anti-pattern. I would recommend instead changing `AppHeader`'s function signature to make it be a true React component (i.e., accepts a `props` object that may be destructured), rather than the existing function:

```jsx
const AppHeader = ({ courseNumber, courseOrg, courseTitle, courseId }) => (...);
```

Reply:
Sure, I will update it